app/controllers/buildings_controller.rb

```diff
@@ -144,4 +181,11 @@ def add_classrooms_for_building(bldrecnbr)
         note = " API returned bo data about classrooms for the building"
       end
     end
+
+    def delete_building(building)
+      Resource.where(room_id: building.rooms.ids).delete_all
+      Room.where(floor_id: building.floors.ids).delete_all
+      Floor.where(building_id: building).delete_all
+      building.delete ? true : false
+    end
```

Comment:
Do we also want to delete Specific Attibutes?

Reply:
Of course, we do. Done here [f5a0223](https://github.com/lsa-mis/room_ready/pull/118/commits/f5a0223d8e48680f2ffb962929309b53c6801c24)